sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -148,7 +148,7 @@ using (FileStream stream = new FileStream(receiptPath, FileMode.Open))
 ### Recognize Content
```

Comment:
I was talking to @annelo-msft , and we want to highlight the examples in the readme in this order (we prioritize the ones the customers are more likely to use): recognize custom forms, recognize content, recognize receipts, train model, manage model.
This is different than the order we want in samples (in samples we want to prioritize the ones that are already ready to use): recognize content, recognize receipts, recognize custom forms, train models, manage models, then the "advanced" samples

Reply:
Will move recognize content first